fuzz/fuzz_test.go

```diff
@@ -0,0 +1,15 @@
+package fuzz
+
```

Comment:
Can we rename this file to be dicom_fuzz_test.go?

Reply:
Done